### PR TITLE
Add more currencies to wallet, fix some internal explorer issues

### PR DESCRIPTION
**Pull request by Koochr**

### CODE REVIEW

File: internal/explorer.point/public/index.zhtml [hunk 1]
```diff
@@ -4,10 +4,6 @@
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
-        <meta
```
Comment (tankakatan) [inline internal/explorer.point/public/index.zhtml:7]: Why do we have to delete it?

File: internal/explorer.point/src/App.js [hunk 2]
```diff
@@ -1,72 +1,78 @@
...
-import Zapps from './pages/Zapps'
+import { ProvideAppContext } from "./context/AppContext";
+import { Route, Switch } from "wouter";
+import Header from "./components/Header";
+import Home from "./pages/Home";
+import Identities from "./pages/Identities";
+import Identity from "./pages/Identity";
+import Final from "./pages/Final";
+import Wallet from "./pages/Wallet";
+import Zapps from "./pages/Zapps";
```
Comment (tankakatan) [inline internal/explorer.point/src/App.js:10]: Why do we use double quotes?

File: internal/explorer.point/src/context/AppContext.js [hunk 3]
```diff
@@ -1,45 +1,57 @@
...
-  return <AppContext.Provider value={ context }>{ children }</AppContext.Provider>
-}
+export const useAppContext = () => useContext(AppContext);
+
+export const ProvideAppContext = ({ children }) => {
+    const [walletIdentity, setWalletIdentity] = useState();
+    const [walletError, setWallerError] = useState();
+    const [, setLocation] = useLocation();
+
+    const fetchData = async () => {
```
Comment (tankakatan) [inline internal/explorer.point/src/context/AppContext.js:25]: `useCallback` is missing

File: internal/explorer.point/src/pages/Identities.jsx [hunk 4]
```diff
@@ -1,82 +1,116 @@
...
-        ? 
-        await window.point.contract.events(
-        {host: '@', contract: 'Identity', event: 'IdentityRegistered', filter: {
-          identityOwner: owner
-        }}) 
-        :
-        await window.point.contract.events(
-          {host: '@', contract: 'Identity', event: 'IdentityRegistered'})
-        ;
+    const fetchIdentities = async () => {
```
Comment (tankakatan) [inline internal/explorer.point/src/pages/Identities.jsx:16]: `useCallback` is missing

File: internal/explorer.point/src/pages/Identities.jsx [hunk 5]
```diff
@@ -1,82 +1,116 @@
...
 import { Link } from "wouter";
 import orderBy from "lodash/orderBy";
 
-export default function Identities({owner}) {
-  
-  const [identities, setIdentities] = useState([])
-  const [ikvset, setIkvset] = useState([])
-  const [isLoading, setIsLoading] = useState(true)
+export default function Identities({ owner }) {
+    const [identities, setIdentities] = useState([]);
```
Comment (tankakatan) [inline internal/explorer.point/src/pages/Identities.jsx:8]: Please avoid stateful components, use context instead

**Review by tankakatan — CHANGES_REQUESTED** [review]
No double quotes please.

File: internal/explorer.point/src/pages/Identities.jsx (hunk 5 above)
Comment (Koochr) [reply internal/explorer.point/src/pages/Identities.jsx:8]: Same as above

File: internal/explorer.point/public/index.zhtml (hunk 1 above)
Comment (Koochr) [reply internal/explorer.point/public/index.zhtml:7]: It was a useless tag with non-standard `name` property and placeholder content: https://www.w3schools.com/tags/att_meta_name.asp

**Pull request merged**